src/com/spikes2212/Cameras/CameraController.java

```diff
@@ -0,0 +1,63 @@
+package com.spikes2212.Cameras;
+
+import java.util.Optional;
+
+import com.ni.vision.NIVision.Image;
+
+import edu.wpi.first.wpilibj.vision.USBCamera;
+
+public class CameraController {
+	public static int resolutionWidth = 100, resolutionHeight = 100;
+	private Optional<USBCamera> camera;
+	private boolean on;
+
+	public CameraController(String port, int fps) {
+		try {
+			USBCamera camera = new USBCamera(port);
+			camera.setFPS(fps);
+			camera.setSize(resolutionWidth, resolutionHeight);
+			camera.setExposureAuto();
+			camera.updateSettings();
+			this.camera = Optional.of(camera);
+		} catch (Exception e) {
+			e.printStackTrace();
+			this.camera = Optional.empty();
+		}
+	}
+
+	public void start() {
+		if (camera.isPresent()) {
+			try {
+				camera.get().startCapture();
+				on = true;
+			} catch (Exception e) {
+				e.printStackTrace();
+			}
+		}
+	}
+
+	public void stop() {
+		try {
+			camera.ifPresent(USBCamera::stopCapture);
+		} catch (Exception e) {
+			e.printStackTrace();
+		}
+		on = false;
+	}
+
+	public void getImage(Image image) {

```

Comment:
add check to avoid reading from camera if it's turned of, instead of unnecessary try-catch